apps/root/src/state/aggregator-settings/reducer.ts

```diff
@@ -56,6 +58,32 @@ export default createReducer(initialState, (builder) => {
     .addCase(setPermit2, (state, { payload }) => {
       state.isPermit2Enabled = payload;
     })
+    .addCase(hydrateAggregatorSettings, (state, { payload }) => {
+      if (!isUndefined(payload.confettiParticleCount)) {
```

Comment:
Isn't confetti now fixed for everyone? Or you keep this for old users

Reply:
Old users + if we later change it and make it configurable again, its already there